active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+###  Adjust target offset for sticky headers
+
+This example shows a list control that supports sticky headers. It listens to the `ScrollingView`'s BringingIntoView event to adjust the target vertical offset to account for the sticky header's presence which the `ScrollingView` is unaware of.
+
+*`ScrollingView` (new)*
+
+```csharp
+private void ScrollingView_BringingIntoView(ScrollingView scrollingView, ScrollingBringingIntoViewEventArgs e)
+{
+    // Adjust the BringIntoViewRequestedEventArgs, e.RequestEventArgs, based on provided target vertical
+    // offset and the sticky header curves.
+    AdjustOffsetForStickyHeader(e.RequestEventArgs, e.TargetVerticalOffset);
+}
+```
+
+*ScrollViewer (existing)*
+
+Not supported
+
+### Detect end of bring-into-view contribution
+
+This example shows how you can know when a bring-into-view contribution starts and ends in order to temporarily turn off user interactions during the contribution.
+
+*`ScrollingView` (new)*
+
+```csharp
+private void ScrollingView_BringingIntoView(ScrollingView scrollingView, ScrollingBringingIntoViewEventArgs e)
+{
+    _bringIntoViewScrollInfo = e.ScrollInfo;
+    scrollingView.IgnoredInputKind = ScrollingInputKinds.All;
+}
+
+private void ScrollingView_ScrollCompleted(ScrollingView scrollingView, ScrollingScrollCompletedEventArgs e)
+{
+    if (_bringIntoViewScrollInfo == e.ScrollInfo)
+    {
+        scrollingView.IgnoredInputKind = ScrollingInputKinds.None;
+    }
+}
+```
+
+*ScrollViewer (existing)*
+
+Not supported
+
+
+## Customize the anchor selection
+
+The anchor element in a `ScrollingView` is used to maintain the content of the view port when the scrolling content changes. For example in a list, the center item's position on screen can be maintained when items earlier in the list are removed.
+
+The `ScrollingView` raises its AnchorRequested event to allow handlers to influence the anchor selection.
+
+### Set an explicit anchor element
+
+This example shows a Grid in a `ScrollingView`, and uses a custom algorithm to select an anchor based on an internal currency element in the Grid.
+
+*`ScrollingView` (new)*
+
+```csharp
+private void ScrollingView_AnchorRequested(ScrollingView scrollingView, ScrollingAnchorRequestedEventArgs e)
+{
+    e.Anchor = SelectAnchorElement();
+}
+```
+
+*ScrollViewer (existing)*
+
+Same as above
+
+### Populate the anchor candidates collection
+
+This example shows an application with a vertical StackPanel in a `ScrollingView`. The AnchorCandidates collection is populated with all the children in the StackPanel and the `ScrollingView` picks an anchor based on the VerticalAnchorRatio property.
+
+*`ScrollingView` (new)*
+
+```csharp
+private void ScrollingView_AnchorRequested(ScrollingView scrollingView, ScrollingAnchorRequestedEventArgs e)
+{
+    foreach (UIElement child in _stackPanel.Children)
+    {
+        e.AnchorCandidates.Add(child);
```

Comment:
Similar to above, `e.AddAnchorCandidate(child)`

Reply:
e.AnchorCandidates is a Windows.Foundation.Collections.IVector<Windows.UI.Xaml.UIElement>. We don't want to replicate the IVector object model.
We're copying this WUX existing object model: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.anchorrequestedeventargs.anchorcandidates